tests/changelog.spec.js

```diff
@@ -0,0 +1,55 @@
+import { test, expect } from "@playwright/test";
+const AxeBuilder = require("@axe-core/playwright").default;
+
+// Changelog Page
+test("Changelog has title", async ({ page }) => {
+  await page.goto("/changelog");
+  await expect(page).toHaveTitle("LinkFree user changelog");
+});
+
+// Navigating to Changelog Page
+test("Navigate to Changelog", async ({ page }) => {
+  await page.goto("/roadmap");
+  await page.getByRole("link", { name: "See full list" }).click();
+  await expect(page.locator("h1")).toHaveText("Changelog");
+});
+
+// Changelog Listed
+test("Changelog listed", async ({ page }) => {
+  await page.goto("/changelog");
+  await expect(page.locator("li")).toBeGreaterThan(1);
```

Comment:
Now the test is running it is failing. Could be because `li` has multiple results on the page

This might help
```suggestion
  await expect(page.locator("main li")).toBeGreaterThan(1);
```

Reply:
why have you written `main li` just wanted to know ?